app/src/main/res/layout-sw600dp/activity_main.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/parent_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <fragment
+        android:id="@+id/face_fragment"
+        android:name="com.willowtree.vocable.facetracking.FaceTrackFragment"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/colorPrimaryDark" />
+
+    <FrameLayout
```

Comment:
Rather than declare both a fragment tag and a FrameLayout for the container, you could use a [FragmentContainerView](https://developer.android.com/reference/androidx/fragment/app/FragmentContainerView)

Reply:
Awesome, I wasn't familiar with those!